components/brave_ads/core/internal/database/database_manager.h

```diff
@@ -30,6 +40,11 @@ class DatabaseManager final {
   // Create or open the database.
   void CreateOrOpen(ResultCallback callback);
 
+  // Run a database transaction. The callback takes one argument -
+  // `mojom::DBTransactionResultInfoPtr` containing the info of the transaction.
+  void RunDBTransaction(mojom::DBTransactionInfoPtr mojom_db_transaction,
```

Comment:
nit: Remove DB as no longer needed, i.e., `RunTransaction`

Reply:
I would go with RunDBTransaction here because it is consistent with `mojom::DBTransactionInfoPtr` argument. We also have the similar member in Database class: `Database::RunDBTransaction`.

Reply:
Sounds good I will raise another issue to remove db throughout at some point. Thanks